src/signon.js

```diff
@@ -155,6 +158,8 @@ export async function login({
       return sendPasswordlessLink({ email });
     case "link":
       return loginWithLink({ token, uuid, redirect });
+    case "securityCode":
+      return loginWithSecurityCode({ to, securityCode, redirect });
```

Comment:
I added a convenience method to `loginWithSecurityCode` in case the developer wants to use it:

```js
await login({
  method: "securityCode",
  to: "+15558675309",
  securityCode: "123456",
  redirect: '/custom-path'
});
```